frontend/test/pytest/python_interface/conftest.py

```diff
@@ -104,6 +105,11 @@ def run_filecheck():
             generic program string back to an xDSL module. ``False`` by default.
     """
     if not deps_available:
+        warn(
+            "The filecheck Python package must be installed to use the 'run_filecheck' "
+            "fixture. Install it using 'python -m pip install filecheck'.",
+            UserWarning,
+        )
```

Comment:
This warning was added so that filecheck being missing is slightly more visible.